src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -924,6 +924,24 @@
                 "trap_ids": "ip2me",
                 "trap_group": "queue1_group1"
             }
+        },
+        "LLDP": {
```

Comment:
BTW what is the source of this config?, this config is not listed on https://github.com/Azure/SONiC/wiki/Configuration.

Reply:
LLDP parameters were directly configured via lldpcli. To support north bound interfaces,these configs were added by Broadcom to configure LLDP global and per port parameters and store the same in Config DB.